content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,318 @@
+The cooperation between JavaScript and the web has been a standard. There are different programming languages used for web development, with a notable mention being Python.  Python has micro-frameworks such as `Flask`, `Hug`, and `Cherrypy` which have enabled Python to cooperate with the web.
+
+Instead of using Python micro-frameworks to run the client-side web applications, we have `Brython`.  `Brython` stands for Browser Python.  `Brython` is a client-side scripting language with Python3 execution that is adjusted to the HTML5 environment. 
```

Comment:
```suggestion
Instead of using Python micro-frameworks to run the client-side web applications, we have `Brython`. `Brython` stands for Browser Python. `Brython` is a client-side scripting language with Python3 execution that is adjusted to the HTML5 environment. 
```